### PR TITLE
Unpin miniconda to latest

### PR DESCRIPTION
**Pull request by bjlittle**
Unpin versions of `miniconda` in the `.travis-ci` to be the latest.

### CODE REVIEW

**Comment by bjlittle** [comment]
Thanks @ajdawson.

File: .travis.yml [hunk 1]
```diff
@@ -29,9 +29,9 @@ install:
   # -----------------
   - export CONDA_BASE=http://repo.continuum.io/miniconda/Miniconda
```
Comment (ocefpaf) [inline .travis.yml]: This should be `https` now. (Recent change.)

Comment (bjlittle) [inline .travis.yml]: @ocefpaf Is `http` being deprecated? i.e. we should move to `https`?

Comment (ocefpaf) [inline .travis.yml]: Yes. In fact I am surprised that the `wget` worked here. I spent my whole weekend fixing that in mane places in conda-forge :smile:

Comment (bjlittle) [inline .travis.yml]: You sure know how to have fun :wink: 

Thanks for the heads-up @ocefpaf

Comment (ajdawson) [inline .travis.yml]: It auto-redirects to the https version, which wget seems OK with... Best move to https anyway though.

**Pull request merged**